src/components/Shared/Tables/SimpleTableComponent.tsx

```diff
@@ -1,20 +1,36 @@
 import * as React from "react";
 import {observer} from "mobx-react";
-import {Cell, Column, Table, SelectionModes, RenderMode} from "@blueprintjs/table";
+import {Cell, Column, Table, SelectionModes, RenderMode, RowHeaderCell} from "@blueprintjs/table";
 import {CARTA} from "carta-protobuf";
-import {ProcessedColumnData} from "models";
+//import {ProcessedColumnData} from "models";
 
 export class SimpleTableComponentProps {
-    dataset: Map<number, ProcessedColumnData>;
+    dataset: Map<number, any>;
     columnHeaders: Array<CARTA.CatalogHeader>;
     numVisibleRows: number;
-    columnWidths?: Array<number>;
+    defaultColumnWidths?: Array<number>;
+    defaultRowHeight?: number;
+    enableGhostCells?: boolean;
+    isIndexZero?: boolean;
     updateTableRef?: (ref: Table) => void;
 }
 
 @observer
 export class SimpleTableComponent extends React.Component<SimpleTableComponentProps> {
-    private renderDataColumn(columnName: string, columnData: any) {
+    private widths = this.props.defaultColumnWidths;
```

Comment:
does this work properly? shouldn't this be done in `constructor`?

Reply:
I think both works, but more straightforward to initialize it in constructor. I will adjust this part.

Reply:
moved the variables to CursorInfoComponent.tsx so the issue no longer exists